kernelci.org/content/en/labs/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Labs"
+date: 2024-07-03
+description: "Connecting labs to KernelCI"
+---
+
+Anyone can connect their labs to KernelCI. Once a Lab is connected to KernelCI, it can execute KernelCI tests.
+
+Whether the Lab will receive specific test configurations or just a general guideline of what tests to run is depended on each Labs preference. If the lab is a LAVA lab, for example, the KernelCI [LAVA runtime](../api_pipeline/pipeline/connecting-lab) will generate test jobs for each single test that is configured to run. On the other hand, if labs are implementing their own runtime (or bridge to Maestro APIs) they may choose to generate the individual test jobs themselves.
```

Comment:
Perhaps my understanding of the first sentence is wrong, so don't blindly accept my suggestion :P

```suggestion
It is up to each lab to define whether it will receive specific test configurations or a generic range of test jobs. If the lab is a LAVA lab, for example, the KernelCI [LAVA runtime](../api_pipeline/pipeline/connecting-lab) will generate test jobs for each single test that is configured to run. On the other hand, if a lab is implementing its own runtime (or bridge to Maestro APIs), they may choose to generate the individual test jobs themselves.
```